src/cwe_checker_lib/src/analysis/expression_propagation/tests.rs

```diff
@@ -37,13 +33,13 @@ fn mock_project() -> Project {
             defs: vec![
                 Def::assign(
                     "entry_jmp_def_1",
-                    Variable::mock("X", 8),
-                    Expression::var("Z", 8).un_op(UnOpType::BoolNegate),
+                    variable!("X:8"),
+                    expr!("Z:8").un_op(UnOpType::BoolNegate),
                 ),
                 Def::assign(
                     "entry_jmp_def_2",
-                    Variable::mock("Z", 8),
-                    Expression::var("Z", 8).un_op(UnOpType::IntNegate),
+                    variable!("Z:8"),
+                    expr!("Z:8").un_op(UnOpType::IntNegate),
                 ),
```

Comment:
Considering how often `IntNegate` appears in this test file, it seems worthwhile to teach it to the expression plugin. This would allow us to use the `defs!` makro for all the lists that do noch contain the rare `BoolNegate` in this test file